modules/rn_3_11_0.adoc

```diff
@@ -130,12 +135,92 @@ This API endpoint can be used with `POST`, `GET`, and `DELETE` calls to create,
+* link:https://issues.redhat.com/browse/PROJQUAY-6810[*PROJQUAY-6810*]. The new UI should highlight the tag name with tag icon in logs page
+* link:https://issues.redhat.com/browse/PROJQUAY-6808[*PROJQUAY-6808*]. The new UI can't click the robot account to show credentials in logs page
+* link:https://issues.redhat.com/browse/PROJQUAY-6807[*PROJQUAY-6807*]. The new UI can't see the operations types in log page when quay is in dark mode
+* link:https://issues.redhat.com/browse/PROJQUAY-6770[*PROJQUAY-6770*]. The new UI build image by uploading Docker file should support .tar.gz or .zip
+* link:https://issues.redhat.com/browse/PROJQUAY-6769[*PROJQUAY-6769*]. The new UI should not display message "Trigger setup has already been completed" after build trigger setup completed
+* link:https://issues.redhat.com/browse/PROJQUAY-6768[*PROJQUAY-6768*]. The new UI can't navigate back to current image repo from image build
+* link:https://issues.redhat.com/browse/PROJQUAY-6767[*PROJQUAY-6767*]. The new UI can't download build logs
+* link:https://issues.redhat.com/browse/PROJQUAY-6758[*PROJQUAY-6758*]. The new UI should display correct operation number when hover over different operation type
+* link:https://issues.redhat.com/browse/PROJQUAY-6757[*PROJQUAY-6757*]. The new UI usage log should display the tag expiration time as date format
+//* link:[**].
+
+[id="upgrading-38-311-limitation"]
+=== Upgrading {productname-ocp} 3.8 directly to 3.11 limitation
 
+Upgrading {productname-ocp} from 3.8 to 3.11 does not work. Users must upgrade from {productname-ocp} from 3.8 to 3.9 or 3.10, and then proceed with the upgrade to 3.11. 
 
-[id="limitations-311"]
-=== {productname} 3.11 limitations
+For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html-single/upgrade_red_hat_quay/index#upgrade_overview[Upgrade {productname}].
+
+[id="configurable-resource-limitation"]
+=== Configurable resource request limitation
+
+Attempting to set resource limitations for the `Quay` pod too low results in the pod being unable to boot up with the following statuses returned: `OOMKILLED` and `CrashLoopBackOff`. Resource limitations can not be set lower than the minimum requirement, which can be found on the link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html-single/deploying_the_red_hat_quay_operator_on_openshift_container_platform/index#configuring-resources-managed-components[Configuring resources for managed components on {ocp}] page.
+
+[id="tag-expiration-known-issue"]
+=== Tag expiration known issue
+
+A known issue was discovered when setting expiration time on a tag that is *older* than two years. When this is done, the tag is expired immediately and subsequently garbage collected. It does not end up in the time machine, and is deleted from the registry. This issue occurs because, by default, the `LABELED_EXPIRATION_MAXIMUM` parameter is set to `104w`, or two years.
+
```

Comment:
We observe the behavior is different, by default, if set expiration longer than 2 yeas, it will change to 2 years automatically
https://redhat-internal.slack.com/archives/G7VFPAY7Q/p1710914407076379?thread_ts=1710336827.793269&cid=G7VFPAY7Q

Reply:
@SeanZhao-redhat I'm not sure if I should document this. For the time being I have marked it out so that it is not included in the 3.11 docs.